.travis.yml

```diff
@@ -10,6 +10,12 @@ services:
   - mongodb
   - rabbitmq
   - docker
+before_install:
+  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
```

Comment:
I have a doubt here if we are adding this package, will it be necessary to add the same in the docker image?
@WesleyCharlesBlake ping!

Reply:
good call @wzalazar — wondering the same now!

Reply:
@wzalazar @lautarodragan [https://github.com/poetapp/node/blob/master/Dockerfile#L3] the only packages being installed from apt in the Docker image is `rsync`.

Do we need to include the PPA `ppa:ubuntu-toolchain-r/test` ? what are we installing from this PPA? is it only `gcc-5 g++-5` ?

If this is a dep, we can build it into the Docker image by adding directives to the Dockerfile

Reply:
@WesleyCharlesBlake 
> Do we need to include the PPA ppa:ubuntu-toolchain-r/test ? what are we installing from this PPA? is it only gcc-5 g++-5 ?

yes, is a dependency for jsondl

for more context, I left this issue
https://github.com/digitalbazaar/jsonld.js/issues/254